en/docs/chapter_hashing/hash_map.md

```diff
@@ -1,30 +1,30 @@
 # Hash table
 
-A <u>hash table</u> achieves efficient element querying by establishing a mapping between keys and values. Specifically, when we input a `key` into the hash table, we can retrieve the corresponding `value` in $O(1)$ time.
+A <u>hash table</u>, also known as a <u>hash map</u>, is a data structure that establishes a mapping between keys and values, enabling efficient element retrieval. Specifically, when we input a `key` into the hash table, we can retrive the corresponding `value` in $O(1)$ time complexity.
 
-As shown in the figure below, given $n$ students, each with two pieces of data: "name" and "student number". If we want to implement a query feature that returns the corresponding name when given a student number, we can use the hash table shown in the figure below.
+As shown in the figure below, given $n$ students, each student has two data fields: "Name" and "Student ID". If we want to implement a query function that takes a student ID as input and returns the corresponding name, we can use the hash table shown in the figure below.
 
 ![Abstract representation of a hash table](hash_map.assets/hash_table_lookup.png)
 
-Apart from hash tables, arrays and linked lists can also be used to implement querying functions. Their efficiency is compared in the table below.
+In addition to hash tables, arrays and linked lists can also be used to implement query functionality, but the time complexity is different. The table below shows the comparison of time efficiency for common operations among them:
```

Comment:
I prefer the original translation, also, "for common operations among them" is not a correct syntax.

Reply:
Suggestion taken! Syntax is correct, but seems a little bit wordy.